Runtime/Rigs/XRSetupBase.cs

```diff
@@ -13,7 +14,19 @@ public XRSetupBase()
             IsPrefabMissing = Resources.Load(PrefabName) == null;
             Resources.UnloadUnusedAssets();
         }
-        
+
+        /// <summary>
+        /// Makes sure the XR loader and subsystem is set up.
+        /// </summary>
+        public override void PreSetup()
+        {
+            if (XRGeneralSettings.Instance.Manager.activeLoader == null)
+            {
+                XRGeneralSettings.Instance.Manager.InitializeLoaderSync();
+                XRGeneralSettings.Instance.Manager.StartSubsystems();
+            }
```

Comment:
This fails if the `XR Management Plugin is not installed` (could be the case if the user has the legacy VR support enabled for OpenVR).

Reply:
Additionally, this does not address the same issue for (the legacy) OpenVR.